ow_plexil/src/plans/TestArmMoveCartesian.plp

```diff
@@ -14,29 +14,22 @@ TestArmMoveCartesian:
   // ow_lander/scripts/arm_move_cartesian.py.  These are not all the
   // defaults, just those useful in this plan.
 
-  Real DefaultXYZ = 0; // Position, Euler and Quaternion orientation
-  Real DefaultW = 1;
-
   log_info ("Beginning TestMoveCartesian...");
 
   LibraryCall ArmUnstow();
 
   // Puts scoop upside-down facing away from the lander.
   LibraryCall ArmMoveCartesian (Frame = BASE_LINK,
                                 Relative = false,
-                                X = 1.7, Y = -0.2, Z = 0.6,
-                                OrientX = DefaultXYZ,
-                                OrientY = DefaultXYZ,
-                                OrientZ = DefaultXYZ);
+                                Position = #(1.7 -0.2 0.6),
+                                Orientation = #(0 0 0));
 
   // Puts scoop right side up facing away from the lander.
   LibraryCall ArmMoveCartesian_Q (Frame = BASE_LINK,
                                   Relative = false,
-                                  X = 1.7, Y = -0.2, Z = 0.6,
-                                  OrientX = 1,
-                                  OrientY = 0,
-                                  OrientZ = 0,
-                                  OrientW = 0);
+                                  Position = #(1.7 -0.2 0.6),
+                                  // Should the following be (0 0 0 1)?
+                                  Orientation = #(1 0 0 0));
```

Comment:
I changed the orientation to the value in the comment because that is the most common value I see used (0 0 0 1). The movement looks fine to me.  I'm not sure if the scoop orientation matters for this test.